### PR TITLE
remove usage of j.tools.async.wrappers.sync

### PR DESCRIPTION
**Pull request by zaibon**
MAKE SUR ALL TEST PASSES BEFORE MERGING THIS

#### What this PR resolves:
remove the need of using `j.tools.async.wrappers.sync` hack in the services actions

### CODE REVIEW

File: templates/cloudinit/actions.py [hunk 1]
```diff
@@ -55,8 +55,4 @@ def watchdog_handler(job):
     loop = j.atyourservice.server.loop
     gateway = job.service.parent.consumers['gateway'][0]
     if gateway.model.data.status == 'running':
-        asyncio.ensure_future(job.service.executeAction('start', context=job.context), loop=loop)
-
-
-def monitor(job):
-    pass
+        asyncio.ensure_future(job.service.ayncExecuteAction('start', context=job.context), loop=loop)
```
Comment (rkhamis) [inline templates/cloudinit/actions.py]: Why not execute with job.service.executeAction?

File: templates/container/actions.py [hunk 2]
```diff
@@ -270,4 +269,4 @@ def watchdog_handler(job):
 
     etcd_cluster = etcd[0].consumers.get('etcd_cluster')
     if etcd_cluster:
-        asyncio.ensure_future(etcd_cluster[0].executeAction('watchdog_handler', context=job.context), loop=loop)
\ No newline at end of file
+        asyncio.ensure_future(etcd_cluster[0].asyncExecuteAction('watchdog_handler', context=job.context), loop=loop)
```
Comment (rkhamis) [inline templates/container/actions.py]: Same

File: templates/etcd/actions.py [hunk 3]
```diff
@@ -39,7 +39,7 @@ def watchdog_handler(job):
     loop = j.atyourservice.server.loop
     etcd_cluster = service.consumers.get('etcd_cluster')
     if etcd_cluster:
-        asyncio.ensure_future(etcd_cluster[0].executeAction('watchdog_handler', context=job.context), loop=loop)
+        asyncio.ensure_future(etcd_cluster[0].asyncExecuteAction('watchdog_handler', context=job.context), loop=loop)
```
Comment (rkhamis) [inline templates/etcd/actions.py]: same

File: templates/cloudinit/actions.py (hunk 1 above)
Comment (zaibon) [reply templates/cloudinit/actions.py]: cause this is the watch_dog handler, which need to be async, right ?

File: templates/container/actions.py (hunk 2 above)
Comment (zaibon) [reply templates/container/actions.py]: Same

File: templates/etcd/actions.py (hunk 3 above)
Comment (zaibon) [reply templates/etcd/actions.py]: same

File: templates/node.zero-os/actions.py [hunk 4]
```diff
@@ -282,17 +275,15 @@ def uninstall(job):
     service = job.service
     stats_collector_service = get_stats_collector(service)
     if stats_collector_service:
-        j.tools.async.wrappers.sync(stats_collector_service.executeAction(
-            'uninstall', context=job.context))
+        stats_collector_service.executeAction('uninstall', context=job.context))
```
Comment (serboctor) [inline templates/node.zero-os/actions.py]: This line contains a syntax error which pretty much means none of these changes has been tested.

File: templates/node.zero-os/actions.py [hunk 5]
```diff
@@ -282,17 +275,15 @@ def uninstall(job):
...
 
     statsdb_service = get_statsdb(service)
     if statsdb_service and str(statsdb_service.parent) == str(service):
-        j.tools.async.wrappers.sync(statsdb_service.executeAction(
-            'uninstall', context=job.context))
+        statsdb_service.executeAction'uninstall', context=job.context))
```
Comment (serboctor) [inline templates/node.zero-os/actions.py]: Another syntax error.

**Comment by zaibon** [comment]
will be merge from https://github.com/zero-os/0-orchestrator/tree/disaster_recovery

**Pull request closed without merging**